designs/virtualization/4.6/Upload-image-prior-to-creating-VM/Upload Image prior to creating vm.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Upload Image (data source) prior to creating a VM
+
+This PR deals with when a user wants to upload an image (data source) prior to creating a virtual machine.
+It needs to be accomplished via the storage/PVC area.
+It should ensure the user understands what the backing PVC does but does not remove the Disk terminology.
+
+The ‘Create PVC’ button will change into a dropdown, so users will be able to select ‘Upload data to Persistent Volume Claim’ from this dropdown.
+That will make 2 separate actions for creation, one of them being able to 'Upload data to Persistent Volume Claim'.
+By creating a dropdown and adding this new action, we'll highlight this action as a new feature (in an existing form) so the users will be able to easily find it.
+
+![PVC list with drop down button](img/PVC-dropdown.png)
+
+They will get to a new upload screen with an inline info alert that says this PVC will be created using a DataVolume through Container Data importer (CDI).
+
+The user chooses an image (data source) to upload.
+In order to allow the user to flag an OS for the data they are uploading, they need to select an OS to attach data to. This selection will define the name/namespace of the Persistent Volume Claim.
+If they want it to back an Operaing System they'll check the check box and the Operating System dropdown will be shown. 
+Once that happens the name/namespace fields are filled in and become read only.
+Then the user fills the rest of the PVC details: size, storage class, access mode. Some of the fields can be filled out but editing will be optional as well.
+
+![browse to upload](img/Upload-data-to-pvc-1.png)
+
+![os to attach the data to](img/Upload-data-to-pvc-1-2.png)
```

Comment:
nits:
PVC name * is below the field
Can you confirm that the console uses disabled and not readonly for these inputs that can't be changed.

Reply:
Thanks, sorry. Wrong terminology.

Reply:
I'm looking at the fields and the page feels a little content heavy I'm not sure if it's just the spacing but what if we took the help text and put it in (?) popovers on the labels instead. 
I think help text like this (under the input) is usually meant for formatting type of hints. WDYT?

Reply:
we will also want to update these designs to include the namespace name now too right? like shown [here](https://xd.adobe.com/view/a8f84349-c92a-443e-bb8c-5e965144473d-fdab/)

Reply:
+1 thanks